src/stylesheets/components/_alerts.scss

```diff
@@ -1,3 +1,39 @@
+$alert-slim-icon-size: units(3);
```

Comment:
We'll be adding this icon with a `:before` block and the mask technique instead of a background image on the alert itself, so some vars and calculations need to change.

Reply:
We might need a `$alert-slim-icon-padding`. The icon on slim variation stays the same but the padding varies when the icon size is updated.

For example, change `$theme-alert-icon-size` to `6`.

![image](https://user-images.githubusercontent.com/3385219/110991033-b64ff500-8339-11eb-8bb6-1c33fa48ac14.png)